com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,76 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --allow=bluetooth
+  - --filesystem=host
+  - --filesystem=xdg-run/pipewire-0
+  - --env=XDG_CURRENT_DESKTOP=GNOME
```

Comment:
why setting this?

Reply:
-filesystem=xdg-run/pipewire-0  as I learned from other repos is for Jack server support, app can be configured to use it.
--env=XDG_CURRENT_DESKTOP=GNOME - might be a leftover from previous experiments, will remove it.

Reply:
> -filesystem=xdg-run/pipewire-0

It's only needed if the app is using Pipewire apis directly

Reply:
Fixed

Reply:
> > -filesystem=xdg-run/pipewire-0
> 
> It's only needed if the app is using Pipewire apis directly

It's needed for JACK as it is the only way JACK works: pw-jack